### DISCUSSION TITLE
Scaling?

### DISCUSSION THREAD

**Discussion by minchinweb** [remotely-save/remotely-save#131] [General]
I'm wondering if the plugin has been load tested at scale (like thousands of notes, or tens of thousands of notes), and how it holds up. Any insight is appreciated!

**Comment by fyears** [comment]
I have created a thousand notes and they are properly uploaded to s3, but slowly. Dropbox and OneDrive and Webdav server may or may not have some rate-limit settings on server sides, which cannot be easily tested, or may change dynamically.
